src/containers/PipelineResource/PipelineResource.js

```diff
@@ -138,129 +142,199 @@ export /* istanbul ignore next */ class PipelineResourceContainer extends Compon
     }
 
     const { params, secrets, type } = pipelineResource.spec;
+    let formattedLabelsToRender = [];
+    if (pipelineResource.metadata.labels) {
+      formattedLabelsToRender = formatLabels(pipelineResource.metadata.labels);
+    }
 
     return (
       <>
         <h1>{pipelineResourceName}</h1>
-        <h2>{`Type: ${type}`}</h2>
 
-        <DataTable
-          rows={params.map(({ name, value }) => ({
-            id: name,
-            name,
-            value
-          }))}
-          headers={[
-            {
-              key: 'name',
-              header: intl.formatMessage({
-                id: 'dashboard.tableHeader.name',
-                defaultMessage: 'Name'
-              })
-            },
-            {
-              key: 'value',
-              header: intl.formatMessage({
-                id: 'dashboard.tableHeader.value',
-                defaultMessage: 'Value'
-              })
-            }
-          ]}
-          render={({
-            rows,
-            headers,
-            getHeaderProps,
-            getRowProps,
-            getTableProps
-          }) => (
-            <TableContainer title="Params">
-              <Table {...getTableProps()}>
-                <TableHead>
-                  <TableRow>
-                    {headers.map(header => (
-                      <TableHeader {...getHeaderProps({ header })}>
-                        {header.header}
-                      </TableHeader>
-                    ))}
-                  </TableRow>
-                </TableHead>
-                <TableBody>
-                  {rows.map(row => (
-                    <TableRow {...getRowProps({ row })}>
-                      {row.cells.map(cell => (
-                        <TableCell key={cell.id}>{cell.value}</TableCell>
+        <Tabs selected={0}>
+          <Tab
+            label={intl.formatMessage({
+              id: 'dashboard.resource.detailsTab',
+              defaultMessage: 'Details'
+            })}
+          >
+            <div className="details">
+              <div className="resource--detail-block">
+                <p>
+                  <span>
+                    {intl.formatMessage({
+                      id: 'dashboard.metadata.dateCreated',
+                      defaultMessage: 'Date Created:'
+                    })}
+                  </span>
+                  <FormattedDate
+                    date={pipelineResource.metadata.creationTimestamp}
+                    relative
+                  />
+                </p>
+                <p>
+                  <span>
+                    {intl.formatMessage({
+                      id: 'dashboard.metadata.labels',
+                      defaultMessage: 'Labels:'
+                    })}
+                  </span>
+                  {formattedLabelsToRender.length === 0
```

Comment:
Is the equals 0 check needed here ?

Reply:
unfortunately yes, otherwise we end up rendering a '0' in the UI

Reply:
hrmm I guess not

Reply:
updated